Sources/PackageRegistryTool/PackageRegistryTool+Publish.swift

```diff
@@ -191,17 +181,34 @@ extension SwiftPackageRegistryTool {
                     )
                 }
 
+                var contentPaths: [AbsolutePath] = []
+                var contentSignaturePaths: [AbsolutePath: AbsolutePath] = [:]
+
                 swiftTool.observabilityScope.emit(info: "signing the archive at '\(archivePath)'")
-                let signaturePath = workingDirectory
+                contentPaths.append(archivePath)
+                contentSignaturePaths[archivePath] = workingDirectory
                     .appending("\(self.packageIdentity)-\(self.packageVersion).sig")
-                signature = try PackageArchiveSigner.sign(
-                    archivePath: archivePath,
-                    signaturePath: signaturePath,
+
+                if let metadataPath = self.customMetadataPath {
```

Comment:
does this mean the user must specify the metadata path, or is there still a default location (ie <root>/package-metadata.json) where we look for it if not specified?

Reply:
yes, user must specify path if they want to include metadata, and metadata is no longer required.